src/test/resources/roles.yml

```diff
@@ -1289,6 +1300,20 @@ point_in_time_2:
       allowed_actions:
         - "manage_point_in_time"
 
+point_in_time_3:
```

Comment:
Where is this new role being used?

Reply:
Here: https://github.com/opensearch-project/security/commit/f7cc569c9d3fa5d5432c76c854eed280d45ce6f4#diff-77e2d2460234b7e8faa37a83d4a2eb672d0f5a4c678a229dc3a9c7e11c8d8b4aL785

Reply:
@cwperks Sorry, I don't see how the `point_in_time_3` is used in tests.

Reply:
I see the rolesmapping now. Thanks @cwperks !